### DISCUSSION TITLE
More new models.. DepthFM and Geowizard

### DISCUSSION THREAD

**Discussion by gituser123456789000** [nagadomi/nunif#106] [Ideas]
More new models to check created in the past week or so..

DepthFM
https://depthfm.github.io/

Geowizard
https://fuxiao0719.github.io/projects/geowizard/

**Comment by gituser123456789000** [comment]
Based on their example images and comparison sliders.. they look like they make Depth-Anything and Marigold (and therefor likely every other model) obsolete.

I'm not sure which is better. I'm about to start testing, if I can figure them out.

**Reply by kaelsonofkrypto** [reply]
Geowizard looks like a great model, i second this, i would love if this was added
